### PR TITLE
js file to fill up big db

### PR DESCRIPTION
**Pull request by mogaleaf**
Here is a js file that fills up a simple db with same value in both columns but on with a PK and so Index and one without.
You can separate the insert and the query so if you want you can load it before the class but it's not that long on my computer.
the result is pretty clear :

Execution time with a primary key: 4ms
Execution time with no index: 10678ms

### CODE REVIEW

File: Week2/big-data.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const util = require('util');
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
```
Comment (unmeshvrije) [inline Week2/big-data.js]: could you change the user from user to 'hyfuser' and password to 'hyfpassword' ?

Comment (mogaleaf) [reply Week2/big-data.js]: yes you are right done !

**Comment by mogaleaf** [comment]
I moved it to week 1.
I had open the pr to your branch, do you want me to open to master?

**Comment by unmeshvrije** [comment]
I think it's better. Then I can directly merge it into master.
   Regards,
   Unmesh Joshi

**Comment by mogaleaf** [comment]
done, pr is now on master branch (also removed an unused sleep method)

**Pull request merged**